go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/hashicorp/net-rpc-msgpackrpc v0.0.0-20151116020338-a14192a58a69
 	github.com/hashicorp/raft v1.1.2
 	github.com/hashicorp/raft-boltdb v0.0.0-20171010151810-6e5ba93211ea
-	github.com/hashicorp/serf v0.9.4
+	github.com/hashicorp/serf v0.9.5
```

Comment:
Usually I'll also update the `api/go.mod` reference to serf to match the root module if I have to change one.

Reply:
Since these are minimum versions, not exact version requirements, updating `api/go.mod` shouldn't be necessary unless that package actually makes use of the new things in `serf`.